server/resources/stripe_checkout.py

```diff
@@ -0,0 +1,31 @@
+from flask import request
+from flask_restful import Resource
+from flask_jwt_extended import (
+    create_access_token, jwt_required, get_jwt_identity)
+from models.user_model import UserModel
+import stripe, os
+
+
+stripe.api_key = os.environ.get('STRIPE_SECRET_KEY')
+
+class HandlePayment(Resource):
+    @jwt_required
+    def post(self):
+        try:
+            data = request.get_json()
+            charge = stripe.Charge.create(
+            amount=data['credits']*1000,
+            currency='cad',
+            description='Credits Purchased',
+            source=data['token']['id']
+            )
```

Comment:
formatting, tab line 17-20

Reply:
I am confused with this formatting. Would you please give me a quick formatting sample ?